src/order/filter/basic.lean

```diff
@@ -727,10 +727,10 @@ lemma map_le_iff_le_comap : map m f ≤ g ↔ f ≤ comap m g :=
 lemma gc_map_comap (m : α → β) : galois_connection (map m) (comap m) :=
 assume f g, map_le_iff_le_comap
 
-lemma map_mono (h : f₁ ≤ f₂) : map m f₁ ≤ map m f₂ := (gc_map_comap m).monotone_l h
-lemma monotone_map : monotone (map m) | a b := map_mono
-lemma comap_mono (h : g₁ ≤ g₂) : comap m g₁ ≤ comap m g₂ := (gc_map_comap m).monotone_u h
-lemma monotone_comap : monotone (comap m) | a b := comap_mono
+lemma map_mono : monotone (map m) := (gc_map_comap m).monotone_l
+lemma monotone_map (m : α → β) : monotone (map m) := map_mono
```

Comment:
I don't think you need to keep the two of them. A quick grep tells me that `monotone_map` and `monotone_comap` are just used one, so you can remove them and replace their use with `map_mono` and `comap_mono`.

Reply:
`map_mono.map_inf_le` doesn't work for some reason; it needs `m` to be an explicit argument. What should I do?

Reply:
In the rare cases where you need the `m` explicit, you can use the `@` version.

Reply:
Removed `monotone_(co)map`.